compiler/rustc_middle/src/metadata.rs

```diff
@@ -43,4 +43,6 @@ pub struct ModChild {
     /// Reexport chain linking this module child to its original reexported item.
     /// Empty if the module child is a proper item.
     pub reexport_chain: SmallVec<[Reexport; 2]>,
+    /// Is this an ambiguous binding?
+    pub ambiguity: bool,
```

Comment:
The ideal type would be `﻿NameBindingData`, but this would require moving several structs from `﻿rustc_resolve` to `﻿rustc_middle`. Therefore, we are using a ﻿`bool` as a tag instead.